README.md

```diff
@@ -226,19 +226,21 @@ synthetic data as it is already log2 transformed. For your own data just
 use `dplyr`’s `mutate()` together with `log2()`.
 
 In addition to filtering and log2 transformation it is also advised to
-median normalise your data to equal out small differences in overall
-sample intensities that result from unequal sample concentrations.
-**protti** provides the `median_normalisation()` function for this
-purpose. This function generates an additional column called
+normalise your data to equal out small differences in overall sample
+intensities that result from unequal sample concentrations. **protti**
+provides the `normalise()` function for this purpose. We generally
```

Comment:
Not sure if I'd say that we recommend to use median normalisation. For some applications there's probably better options. I'd maybe say something along the lines of "For this example we will use median normalisation".